### PR TITLE
412 web company inscription

### PR DESCRIPTION
**Pull request by CedricCORGE**
- New system of company creation during register
- Can invite collaborator in your teams during the registration process

https://github.com/user-attachments/assets/0f0b45e7-ce6b-4d28-83aa-8c4276ac2557

### CODE REVIEW

**Review by invpuppet — APPROVED** [review]

**Review by H-chauvet — APPROVED** [review]

File: Server/controllers/Web/organization.js [hunk 1]
```diff
@@ -119,6 +121,29 @@ exports.updateType = (res, organization) => {
       },
     });
   } catch (err) {
-    throw "Something happen while updating organization's type";
+    throw res.__("errorOccured");
+  }
+};
+
+exports.inviteMember = (res, memberList, company) => {
```
Comment (31Nathan) [inline Server/controllers/Web/organization.js:128]: doc

File: Server/controllers/Web/user.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ exports.findUserByEmail = async (res, email) => {
       },
     });
   } catch (err) {
+    console.log("err: " + err);
```
Comment (31Nathan) [inline Server/controllers/Web/user.js:23]: Keep it ?

File: Server/routes/Web/organization.js [hunk 3]
```diff
@@ -126,4 +129,70 @@ router.post("/update-type/:id", async (req, res, next) => {
   }
 });
 
+router.post("/invite-member", async (req, res) => {
```
Comment (31Nathan) [inline Server/routes/Web/organization.js:132]: Route documentation

File: Server/routes/Web/organization.js [hunk 4]
```diff
@@ -126,4 +129,70 @@ router.post("/update-type/:id", async (req, res, next) => {
...
+    res.status(200).send(organization);
+  } catch (err) {
+    if (res.statusCode == 200) {
+      res.status(500);
+    }
+    res.send(err);
+  }
+});
+
+router.post("/add-member", async (req, res) => {
```
Comment (31Nathan) [inline Server/routes/Web/organization.js:162]: Same here

File: Server/routes/Web/user.js [hunk 5]
```diff
@@ -348,6 +346,32 @@ router.post("/update-company/:email", async (req, res, next) => {
   }
 });
 
+router.put("/update-company", async (req, res) => {
```
Comment (31Nathan) [inline Server/routes/Web/user.js:349]: Route doc

**Pull request merged**